cypress.json

```diff
@@ -1 +1,10 @@
-{}
+{
+  "baseUrl": "http://localhost:8100",
+  "env": {
+    "username": "jack",
+    "email": "test@example.com",
+    "password": ""
+  },
+  "viewportWidth": 1400,
```

Comment:
Are these variables being used? If not, eliminate them.

Reply:
Vars removed.

Reply:
Both are still there in the newest commit